src/clustering.cr

```diff
@@ -108,8 +104,9 @@ class Clustering
   # Accepts a block that will be called with cluster nodes
   # during stabilization events
   #
-  def start(&stabilize : Array(HoundDog::Service::Node) ->)
+  def start(on_stable : String -> Nil? = nil, &stabilize : Array(HoundDog::Service::Node) ->)
```

Comment:
That nilable union looks to be binding to the wrong there there. Should be `on_stable : (String -> Nil)? = nil` if it's an optional arg.

Reply:
good catch!